lib/t/feeds/feed_profile.ex

```diff
@@ -10,6 +10,7 @@ defmodule T.Feeds.FeedProfile do
     field :hidden?, :boolean
     field :last_active, :utc_datetime
     field :location, Geo.PostGIS.Geometry
+    field :h3, :decimal
```

Comment:
I thought about it a bit, and I think a [custom ecto type](https://hexdocs.pm/ecto/Ecto.Type.html) with `i64 (bigint)` can be used. Before dumping we'd subtract, before reading we'd add `i64.max`, that'd be equivalent to u64.